### ISSUE TITLE
ImportError: cannot import name 'BloomTokenizer' from 'transformers'

### ISSUE THREAD

**Issue by rhvaz** [huggingface/transformers#18178]
### System Info

transformers==4.20.1
torch==1.12.0
Python 3.9.13
GPU: yes (running on GCP)

### Who can help?

@SaulLu 

### Information

- [X] The official example scripts
- [ ] My own modified scripts

### Tasks

- [ ] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [ ] My own task or dataset (give details below)

### Reproduction

```
from transformers import BloomTokenizer, BloomModel
```

I was following https://huggingface.co/docs/transformers/model_doc/bloom

Specifically 
```
from transformers import BloomTokenizer, BloomModel
import torch

tokenizer = BloomTokenizer.from_pretrained("bigscience/Bloom")
model = BloomModel.from_pretrained("bigscience/Bloom")

inputs = tokenizer("Hello, my dog is cute", return_tensors="pt")
outputs = model(**inputs)

last_hidden_states = outputs.last_hidden_state
```

### Expected behavior

No import error.

**Comment by SaulLu** [comment]
Hi @rhvaz,

I am sincerely sorry that you have encountered this issue. We do have a small typo in our documentation, which has been resolved in the PR https://github.com/huggingface/transformers/pull/18005 but not yet deployed on our website.

In the meantime, here is the snippet that should work:
```python
from transformers import BloomTokenizerFast, BloomModel
import torch

tokenizer = BloomTokenizerFast.from_pretrained("bigscience/Bloom")
model = BloomModel.from_pretrained("bigscience/Bloom")

inputs = tokenizer("Hello, my dog is cute", return_tensors="pt")
outputs = model(**inputs)

last_hidden_states = outputs.last_hidden_state
```

**Comment by SaulLu** [comment]
You spotted another typo in the name of the checkpoint! Here's a new snippet that should work:
```python
from transformers import BloomTokenizerFast, BloomModel
import torch

tokenizer = BloomTokenizerFast.from_pretrained("bigscience/bloom")
model = BloomModel.from_pretrained("bigscience/bloom")

inputs = tokenizer("Hello, my dog is cute", return_tensors="pt")
outputs = model(**inputs)

last_hidden_states = outputs.last_hidden_state
```

I also took this opportunity to share the same fix in the documentation in PR #18187
